models/actions/run.go

```diff
@@ -305,7 +306,7 @@ func InsertRun(ctx context.Context, run *ActionRun, jobs []*jobparser.SingleWork
 		} else {
 			hasWaiting = true
 		}
-		job.Name, _ = util.SplitStringAtByteN(job.Name, 255)
+		job.Name = base.EllipsisStringWholeWord(job.Name, 255)
```

Comment:
I don't think we should do any pre-processing in the backend here.
Why not just hide it in the frontend using CSS?

Reply:
I mean, the new approach looks less buggy than the old one, but still:
Why not let the browser handle it?

Reply:
I don't think it's related to the frontend.

The code here is not for a better UI, but for protecting the database when the job name field is `VARCHAR(255)`, it could cause a failure (for PostgreSQL) or value truncating (for MySQL).

We cannot forbid users to do that since it's parsed from the YAML file the user uploaded.